pkg/apparmor/etc/apparmor.d/abi/3.0

```diff
@@ -0,0 +1,78 @@
+query {label {multi_transaction {yes
```

Comment:
Is this lack of indentation the normal form for this type of files? Isn't human readable.

Reply:
I suppose, AFAIK that file is consumed by the `apparmor_parser` to compile text based rules to binary, the `apparmor.d\abstractions` and `apparmor.d\tunables` are the files used in writing profiles (like include <header> ) and should be human readable.

Reply:
> Are these files copied from somewhere? If so it makes sense to mention that as well.

mentioned already in the commit message : 
```
Content of "pkg/apparmor/etc" are copied from Apparmor
official reposiory located at https://gitlab.com/apparmor/apparmor.
```